src/screens/Settings/profile.module.scss

```diff
@@ -1,10 +1,12 @@
+@import 'styles/layout.scss';
+
 .head {
   display: flex;
   justify-content: flex-start;
-  height: 12rem;
 
   img {
     height: 100%;
```

Comment:
Any reason for keeping 2 `height:` selectors? Isn't the second overwriting the first one?

Reply:
There is no reason, it's just an oversight. I will remove it.